.github/lint

```diff
@@ -15,7 +15,7 @@ if [ "$(head -n 1 $DICTIONARY)" != "# Gboard Dictionary version:1" ]; then
 fi
 
 echo "Checking that remaining lines are valid..."
-INVALID_LINES=$(tail -n +2 $DICTIONARY | grep -v -n -P -e '^(\S*?\t.*?\t)$')
+INVALID_LINES=$(cat $DICTIONARY | grep -v -n -P -e '^(\S*?\t.*?\t)$' | grep -v "1:# Gboard Dictionary version:1")
```

Comment:
Awesome! Thanks for contributing!

Perhaps the `tail` command can be used instead of `grep` to remove the first line at the end? That way it is shorter and doesn't require hard-coding the value.

Additionally, grep can take a file as a parameter, so no need to pipe from cat.

Let me know what you think :)

```suggestion
INVALID_LINES=$(grep -v -n -P -e '^(\S*?\t.*?\t)$' $DICTIONARY | tail -n +2)
```

Reply:
That's actually looks much better.
Thanks 🙂